### PR TITLE
Please review these suggested updates

### PR DESCRIPTION
**Pull request by stevenlemme**
Pushing these for your review per our conversation Dave.
Then merge them

### CODE REVIEW

File: enterprise-manageability-library/Readme.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+1. [EM Environment Setup](enterprise_manager/em_workshop_setup.md)
+2. [Lifecycle Automation](enterprise_manager/em_db_lifecycle_automation.md)
+3. [Find, Fix, and Validate](enterprise_manager/em_find_fix_validate.md)
+
+## Enterprise Manager Labs in PDF 
+1. [Environment Setup](https://stbeehive.oracle.com/content/dav/st/ent-mgr/em_public/training/FY20/DB_Options_roadshow_nov2019/EM_Workshop_Setup.pdf)
+7. [Database Lifecycle Automation Using EM](https://stbeehive.oracle.com/content/dav/st/ent-mgr/em_public/training/FY20/DB_Options_roadshow_nov2019/EM_DB_Lifecycle_Automation.pdf)
+8. [Find, Fix and Validate using EM 13C](https://stbeehive.oracle.com/content/dav/st/ent-mgr/em_public/training/FY20/DB_Options_roadshow_nov2019/EM_Find_Fix_Validate.pdf)
+
+## NOTE: WHILE ITS NICE TO HAVE, PROBABLY BEST TO LINK THEM FROM A PUBLIC ORADOCS INSTEAD OF BEEHIVE BECUASE ANYONE CAN TRY TO ACCESS...
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/Readme.md]: Agreed, let's remove the pdf links for now until we find a suitable external place to link them to.

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+============
+
+>   This lab will show you how to login to the cloud and setup a compute
+>   instance using the custom vm images that has Oracle Enterprise Manager as
+>   well as Oracle Database targets on it.
+
+Lab Assumptions
+===============
+
+-   Each participant has been provided an account on the c4u03 tenancy and the
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: Let's remove "c4u03 tenancy" and mark with "[provided tenancy]"

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    network(VCN) has been pre-created (including ingress rules).
+
+-   Each participant has a public/private key to login to instances post
+    creation
+    ([instructions](https://github.com/kaymalcolm/learning-library/blob/master/data-management-library/database/options/environment-setup-c4u03.md#section-2-create-an-ssh-key-pair))
+
+Create Compute Instance
+=======================
+
+1.  Login to c4u03 using your account, once logged-in Click on the navigation
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: Let's remove "c4u03" and mark with "[provided tenancy]"

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    select the root compartment or any other compartment**.
+
+![](media/1a4bdcea9aeb2f4faa2918ce0de034bb.jpg)
+
+1.  Click the **Create Instance** button.
+
+![](media/67cdb773d62b0246e75ff98fde2dee9d.jpg)
+
+1.  Enter your desired instance name wich should be unique. Suggested name: Use
+    your guid EMworkshop (pchandir_em_workshop). *Do NOT enter name as-is from
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: remove (pchandir_em_workshop)

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+![](media/0510ee95794684fa3d67ae5668fd6818.jpg)
+
+![](media/9a794cdd7f2f6fd7f5ae3f8c0890d8cd.png)
+
+![](media/db534fcc781e9902a4bee140704da5a3.jpg)
+
+1.  In the Configure networking section, select the dboptions compartment and
+    the VCN you created in an earlier section or the default (dboption1105).
+    Click on the radio button to **assign a public address**. This is important.
+    **DO NOT OVERLOOK THIS STEP!!!!!!!**
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: Please make this sentence in red and/or bigger font to stand out.
"Click on the radio button to **assign a public address**. This is important.
    **DO NOT OVERLOOK THIS STEP!!!!!!!**"

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+1.  Click the **Create** button to create your instance. Your instance will be
+    in the provisioning state for about 4 minutes but it will take about 15 mins
+    to start the databases as well as Enterprise Manager on your VM. Verify that
+    you chose the correct image. In a few minutes, you can also verify that you
+    have a public IP address. View the Work Requests at the bottom, this will
+    show where your instance is.
+
+![](media/1ec471ec8adfa265e31cc88e64f6c809.jpg)
+
+1.  Once the instance moves to the Running state, locate your **public IP
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: All the steps numbers are "1."  They need to be sequentially numbered.

File: enterprise-manageability-library/enterprise_manager/em_db_lifecycle_automation.md [hunk 7]
```diff
@@ -0,0 +1,770 @@
+![](media/rdwd-emheader.png) 
+
+Introduction
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_db_lifecycle_automation.md]: For the entire md file -- all code references should be in courier/code font

File: enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md [hunk 8]
```diff
@@ -0,0 +1,1067 @@
+![](media/rdwd-emheader.png) 
+
+#### **Table of Contents**
+
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md]: Make sure all code reference are in courier/code font throughout the file.

File: enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md [hunk 9]
```diff
@@ -0,0 +1,1067 @@
...
+-   Category: Session
+
+-   IP address: Your service instance’s public IP address
+
+-   Port: 22
+
+-   Connection type: SSH
+
+![](media/7c9e4d803ae849daa227b6684705964c.jpg)
+
```
Comment (davidleroyatoracle) [inline enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md]: Top-level steps need to be sequentially numbered . Right now they all say “1.”

File: enterprise-manageability-library/enterprise_manager/em_db_lifecycle_automation.md (hunk 7 above)
Comment (stevenlemme) [reply enterprise-manageability-library/enterprise_manager/em_db_lifecycle_automation.md]: Markdown doesnt really have fonts. After doing some research, their seem to be hacks that might work however they also add a lot of work. So unless you have found a way, I would say to set this "nice to have" aside and focus on the "must haves"

File: enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md (hunk 8 above)
Comment (stevenlemme) [reply enterprise-manageability-library/enterprise_manager/em_find_fix_validate.md]: Markdown doesnt really have fonts. After doing some research, their seem to be hacks that might work however they also add a lot of work. So unless you have found a way, I would say to set this "nice to have" aside and focus on the "must haves"

File: enterprise-manageability-library/enterprise_manager/em_workshop_setup.md (hunk 2 above)
Comment (stevenlemme) [reply enterprise-manageability-library/enterprise_manager/em_workshop_setup.md]: done

**Comment by davidleroyatoracle** [comment]
Good test.

**Pull request closed without merging**